netstandard/ref/System.Net.Security.cs

```diff
@@ -95,6 +96,34 @@ public enum ProtectionLevel
         Sign = 1,
     }
     public delegate bool RemoteCertificateValidationCallback(object sender, System.Security.Cryptography.X509Certificates.X509Certificate certificate, System.Security.Cryptography.X509Certificates.X509Chain chain, System.Net.Security.SslPolicyErrors sslPolicyErrors);
+    public delegate System.Security.Cryptography.X509Certificates.X509Certificate ServerCertificateSelectionCallback(object sender, string hostName);
+    public readonly partial struct SslApplicationProtocol : System.IEquatable<System.Net.Security.SslApplicationProtocol>
+    {
+        public static readonly System.Net.Security.SslApplicationProtocol Http11;
+        public static readonly System.Net.Security.SslApplicationProtocol Http2;
```

Comment:
[HttpVersion](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpversion.version20?view=netcore-2.1&viewFallbackFrom=netframework-4.7.2) is using 20 suffix and I think I saw it at more places, can we make it consistent?

Reply:
I’m against this. Keep this as Http2 as it’s whats used in .NET Core.

Reply:
Also, officially in the RFC, the version is HTTP/2, not 2.0 (on purpose).

Reply:
> [HttpVersion](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpversion.version20?view=netcore-2.1&viewFallbackFrom=netframework-4.7.2) is using 20 suffix and I think I saw it at more places, can we make it consistent?

We cannot change API when we add them to .NET Standard. The only degrees of freedom we have:

1. Add
2. Do not add

We *can* make request for additive changes, but we should separate them from .NET Standard approval. Otherwise, it becomes a multi-dimensional puzzle.